lib/redis.js

```diff
@@ -74,19 +74,32 @@ class RedisClient {
 				try {
 					let command = line.trim();
 					let commands = splitargs(command);
-					let CMD = commands.shift().toLowerCase();
-					let func = this._redis_client[`${CMD}Async`];
-					if (typeof func == "function") {
-						let result = await func.bind(this._redis_client)(...commands);
-						if (Array.isArray(result)) {
-							result.each((item, index) => {
-								console.log("%d) %s", index, item);
+					if (commands.length !== 0) { // we have commands, so process, otherwise just a new prompt
```

Comment:
==0   Should be  >0

Reply:
Humm. `>0` will coerce. `!==0` will not.